src/js/components/Button/README.md

```diff
@@ -260,6 +260,15 @@ Label text to place in the button.
 node
 ```
 
+**normalized**
+
+Whether this is a normalized button. Normalized buttons have consistent 
+      styling regardless of if they contain only an icon or an icon and label.
```

Comment:
I would add a little more description of what will be a good use case to use this prop. this description is not telling much.